### PR TITLE
Issue 926

### PR DESCRIPTION
**Pull request by zmcNotafraid**
For polyjuice creator transaction, return a `account_eth_address `attribute to display

### CODE REVIEW

**Comment by Keith-CY** [comment]
Please add a brief description of this PR so we won't miss it in commit history

**Comment by Keith-CY** [comment]
Since we are now using graphql to fetch a transaction, adding the `created_account` in `polyjuice_creator` field could be more useful

Ref: https://github.com/Magickbase/godwoken_explorer/blob/godwoken-v1-mainnet-prod/docs/schema.graphql#L2400

**Comment by Naupio** [comment]
need rebase branch

**Comment by zmcNotafraid** [comment]
> need rebase branch

We use godwoken-v1-mainnet-prod as base brunch.So no need to rebase

**Comment by zmcNotafraid** [comment]
> Since we are now using graphql to fetch a transaction, adding the `created_account` in `polyjuice_creator` field could be more useful
> 
> Ref: https://github.com/Magickbase/godwoken_explorer/blob/godwoken-v1-mainnet-prod/docs/schema.graphql#L2400

Add created_account

File: docs/schema.graphql [hunk 1]
```diff
@@ -506,7 +506,8 @@ type PolyjuiceCreator {
   hash_type: String
   id: Int
   script_args: String
-  tx_hash: HashFull
+  tx_hash: HashFull,
+  created_account: String
```
Comment (Keith-CY) [inline docs/schema.graphql:510]: it creates the naming convention. We have `from_account` and `to_account` in the schema and they are both of `Account type`, so the `created_account` is expected to be an `Account` too.

Comment (zmcNotafraid) [reply docs/schema.graphql]: It's fixed. Current return 
```
PolyjuiceCreator {
  created_account {
     eth_address
     type
     scirpt_hash
  }
}
```

**Review by Keith-CY — APPROVED** [review]

**Review by Naupio — APPROVED** [review]

**Pull request merged**